src/ui/flightdisplay/QGCFlightDisplay.h

```diff
@@ -69,6 +70,7 @@ class QGCFlightDisplay : public QGCQmlWidgetHolder
     float   latitude            () { return _latitude; }
```

Comment:
Do you know that for these you can just do:
'Q_PROPERTY(float latitude           MEMBER _latitude           NOTIFY latitudeChanged)'
As long as you don't need to reference a latitude() call from C++.